### ISSUE TITLE
svg方式渲染折线图时，当数据源的所有数据值相同时，阴影失效，且影响了线条的展示

### ISSUE THREAD

**Issue by liuhui666** [apache/echarts#12614]
### Version
4.7.0

### Reproduction link
[https://codesandbox.io/s/beautiful-currying-esdpl?file=/src/App.js](https://codesandbox.io/s/beautiful-currying-esdpl?file=/src/App.js)

### Steps to reproduce
打开之后无需操作 直接看到现象
1.   echarts.init(dom, null, {renderer: "svg"});
2. series[0].data =[1,1,1,1,1,1,1]
3. series[0].lineStyle={
            shadowBlur: 20,
            shadowOffsetX: 20,
            shadowOffsetY: 20

}

### What is expected?
能有一条可见的线条

### What is actually happening?
只展示出折线拐点标志，线条没有颜色

---
必要条件是：
1、数据源的所有值一定都要相同
2、lineStyle增加阴影属性

线条的path标签里有style="filter: url("#zr2322-shadow-0");"  导致问题出现

**Comment by Ovilia** [comment]
Thanks for reporting. Seems to be a bug.

**Comment by Ovilia** [comment]
It seems to be a bug with SVG rendering in some Browsers. Tested with Chrome (failed), Firefox (failed), and Safari (OK) under Mac.
Reported at https://bugs.chromium.org/p/chromium/issues/detail?id=1247310#c2

**Comment by ZeyanGuo** [comment]
Same issue at version 5.2.2, is there any solution to fix it?
`
lineStyle: {
  width: 3,
  shadowColor: 'rgba(36, 178, 118, 0.3)',
  shadowBlur: 6,
  shadowOffsetY: 3,
},
`

**Comment by liuhui666** [comment]
这是来自QQ邮箱的自动回复邮件。您好，您的邮件我已收到                                        ——刘慧

**Comment by ZeyanGuo** [comment]
> Same issue at version 5.2.2, is there any solution to fix it? `lineStyle: { width: 3, shadowColor: 'rgba(36, 178, 118, 0.3)', shadowBlur: 6, shadowOffsetY: 3, },`

I changed the renderer method to canvas to fixed this issue.

**Comment by plainheart** [comment]
Yes. This is an issue related to SVG. We will look into it in ecomfe/zrender#792.

**Comment by liuhui666** [comment]
这是来自QQ邮箱的自动回复邮件。您好，您的邮件我已收到                                        ——刘慧

**Comment by liuhui666** [comment]
这是来自QQ邮箱的自动回复邮件。您好，您的邮件我已收到                                        ——刘慧
